content/v1.14/cli/command-reference.md

```diff
@@ -0,0 +1,595 @@
+`--examples-root` as documentation for published packages.
+
+#### Include a runtime image
+
+Functions and Providers require YAML files describing their dependencies and 
+settings as well as a container image for their runtime.
+
+Using `--embed-runtime-image-name` downloads a specified container image and 
+includes the image inside the function or provider package.
+
+The `--embed-runtime-image-tarball` flag includes a local OCI image tarball 
+inside the function or provider package.
+
+
+### xpkg install
+
+Download and install packages into Crossplane with  `crossplane xpkg install`.
+
+By default the `crossplane xpkg install` command uses the Kubernetes 
+configuration defined in `~/.kube/config`.  
+
+Define a custom Kubernetes configuration file location with the environmental 
+variable `KUBECONFIG`.
+
+Specify the package kind, package file and optionally a name to give the package 
+inside Crossplane.
+
+`crossplane xpkg install <package-kind> <registry package name and tag> [<optional-name>`
+
+The package file must be an organization, image and tag on the `xpkg.upbound.io`
+registry on [Upbound Marketplace](https://marketplace.upbound.io/).
+
+For example, to install version 0.42.0 of the 
+[AWS S3 provider](https://marketplace.upbound.io/providers/upbound/provider-aws-s3/v0.42.0):
+
+`crossplane xpkg install provider xpkg.upbound.io/upbound/provider-aws-s3:v0.42.0`
+
+#### Flags
+{{< table "table table-sm table-striped">}}
+| Short flag   | Long flag                                        | Description                                                                                     |
+| ------------ | -------------                                    | ------------------------------                                                                  |
+|              | `--runtime-config=<runtime config name>`         | Install the package with a runtime configuration.                                               |
+| `-m`         | `--manual-activation`                            | Set the `revisionActiviationPolicy` to `Manual`.                                                |
+|              | `--package-pull-secrets=<list of secrets>`       | A comma-seperated list of Kubernetes secrets to use for authenticating to the package registry. |
+| `-r`         | `--revision-history-limit=<number of revisions>` | Set the `revisionHistoryLimit`. Defaults to `1`.                                                |
+| `-w`         | `--wait=<number of seconds>`                     | Number of seconds to wait for a package to install.                                             |
+
+{{< /table >}}
+
+#### Wait for package install
+
+When installing a package the `crossplane xpkg install` command doesn't wait for
+the package to download and install. View any download or installation problems 
+by inspecting the `configuration` with `kubectl describe configuration`.
```

Comment:
this will only be valid if the package is a configuration i think - if it's a provider or function then this command wouldn't find it.

Reply:
If you give the command an invalid Provider package URL it returns immediately and you see 404 errors when you describe the provider.

Is this what you mean or something else?